lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/gateway/ThirdPartyDocumentGateway.java

```diff
@@ -119,10 +119,10 @@ public DocumentCheckResult performDocumentCheck(DrivingPermitForm drivingPermitD
         String drivingPermitDocumentNumber = drivingPermitData.getDrivingLicenceNumber();
         LocalDate drivingPermitIssueDate = drivingPermitData.getIssueDate();
 
-        String dcsEndpointUri = null;
+        String dcsEndpointUri = configurationService.getDcsEndpointUri();
         switch (issuingAuthority) {
             case DVA:
-                dcsEndpointUri = configurationService.getDcsEndpointUri() + "/dva-driving-licence";
+                dcsEndpointUri += "/dva-driving-licence";
```

Comment:
@MMcC-GDS This PR is outdated, can we get this commit onto a new branch and a seperate PR?